src/components/multi-action-button/multi-action-button-classic.style.js

```diff
@@ -0,0 +1,101 @@
+import { css } from 'styled-components';
+import { THEMES } from '../../style/themes';
+import StyledSplitButton from '../split-button/split-button.style';
+import StyledButton from '../button/button.style';
+import StyledSplitButtonChildrenContainer from '../split-button/split-button-children.style';
+
+const getMultiActionButtonClassicStyles = ({ disabled, displayed, theme }) => (theme.name === THEMES.classic ? css`
+  ${StyledSplitButton} > ${StyledButton},
+  ${StyledSplitButton} > ${StyledButton}:focus {
+    margin: 0;
+    height: 31px;
+    padding: 7px 10px 7px 18px;
```

Comment:
could we make this 7px 14px 7px 18px as per my feedback

Reply:
changed